example/.babelrc

```diff
@@ -47,7 +48,8 @@
             "loose": true,
             "useBuiltIns": true
         }],
-        "react"
+        "react",
+        "es2015"
```

Comment:
@doctyper I don't think we need this line because we're using the `env` plugin above. Removing it causes this error though:
```
SyntaxError: Unexpected token import
```
Do you know if we should keep this or not?